csharp-refactoring/RefactoringObjectOrientationAbusers/RefactoringObjectOrientationAbusers/TemporaryField/Incorrect/BookstoreCustomer.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace RefactoringObjectOrientationAbusers.TemporaryField.Incorrect
+{
+    public class BookstoreCustomer
+    {
+        private int discountThreshold;
+        private int boughtBooksCount;
```

Comment:
@zanbor, since these are private fields they need to start with an underscore "_". 
Please fix the code snippets in the article as well. 
Thanks!